cpp/src/io/parquet/page_enc.cu

```diff
@@ -1680,15 +1908,21 @@ __global__ void __launch_bounds__(1)
   // min_values
   encoder.field_list_begin(2, num_pages - first_data_page, ST_FLD_BINARY);
   for (uint32_t page = first_data_page; page < num_pages; page++) {
-    get_extremum(&column_stats[pageidx + page].min_value, col_g.stats_dtype, scratch, &vmin, &lmin);
-    encoder.put_binary(vmin, lmin);
+    auto const min = get_min(&column_stats[pageidx + page].min_value,
+                             col_g.stats_dtype,
+                             scratch,
+                             column_index_truncate_length);
+    encoder.put_binary(min.first, min.second);
```

Comment:
The advantage of using a `std::pair` in `get_min()` is that you can use structured binding to make it more descriptive, should you choose to:
```suggestion
    auto const [min_ptr, min_size] = get_min(&column_stats[pageidx + page].min_value,
                                             col_g.stats_dtype,
                                             scratch,
                                             column_index_truncate_length);
    encoder.put_binary(min_ptr, min_size);
```

Reply:
Implemented.  TIL c++ can do python :)